weave/trace_server/migrations/007_stats_views.down.sql

```diff
@@ -0,0 +1,11 @@
+DROP TABLE IF EXISTS weave_trace_db.calls_merged_stats;
```

Comment:
calls merged is one of the few that we don't need actually

Reply:
still helpful to make counting up usage dramatically faster

Reply:
fair enough - any reason you did not do the other tables?

Reply:
you mean like tables? we only need this for tables we bill on. and besides feedback, no other tables have user-defined fields of arbitrary length

Reply:
feedback does, but i guess we don't bill that right now

Reply:
added one for feedback